### PR TITLE
mzSpecLib Terms Batch 4

### PR DESCRIPTION
**Pull request by mobiusklein**
This batch of terms adds several terms missing from the spreadsheet, and covers the following renamings:
1.  `spectrum position` -> `library entry key` and `spectrum index` -> `library entry index`
2. `library entry class` -> `library spectrum attribute set`

### CODE REVIEW

**Review by edeutsch — COMMENTED** [review]
I'm not certain I'm reading this right, but I think this PR stomps on the previous commit for paper spray ionization MS:1003235 (which I may have misplaced after the MS:4000000 series?)? I'm also wondering why it skips from 71 to 73?

**Comment by mobiusklein** [comment]
I re-arranged terms to be in order of ID, so `paper spray ionization` got moved to before the mzQC terms, which are all still intact.

I can't seem to install `fastobo-validator` from source or binary so I am resorting to using GH actions to find the syntax error.

**Review by edeutsch — COMMENTED** [review]
oh, I see it now, okay. But it's identifier changed from 3235 to 3236. Is that necessary? ThermoeRawFileParser would seem to be using 3235 already:

https://github.com/compomics/ThermoRawFileParser/pull/129/commits/ee973e8ed96254a6b39bd4b102f14da0a993a09b

**Comment by mobiusklein** [comment]
There was a collision from earlier work on the branch, I re-arranged the terms so the original ID is restored for `paper spray ionization`.

**Review by edeutsch — APPROVED** [review]
many thanks, this all looks good to me!

File: psi-ms.obo [hunk 1]
```diff
@@ -22196,6 +22198,68 @@ is_a: MS:1001456 ! analysis software
 is_a: MS:1003207 ! library creation software
 is_a: MS:1002878 ! small molecule analysis software
 
+[Term]
+id: MS:1003234
+name: library entry attribute
+def: "An attribute that describes information about an entry in a sepctral library" [PSI:PI]
```
Comment (henryhlam) [inline psi-ms.obo:22204]: spectral library spelled wrong

File: psi-ms.obo [hunk 2]
```diff
@@ -22196,6 +22198,68 @@ is_a: MS:1001456 ! analysis software
...
+
+[Term]
+id: MS:1003239
+name: library interpretation attribute set
+def: "A set of interpretation-related attributes that is shared by a subset of spectra within the same spectral library" [PSI:PI]
+is_a: MS:1003171 ! spectral library attribute
+
+[Term]
+id: MS:1003240
+name: peak intensity transforma
```
Comment (henryhlam) [inline psi-ms.obo:22233]: should be "transform" not "transforma"

**Pull request merged**